Airports/EGKB/Stars.txt

```diff
@@ -24,20 +24,25 @@ STAR:EGKB:21:SUMUM1C:SUMUM LOGAN JACKO NONVA BABKU LCE21 LCE22 LCE23 RAVSA GAPGI
 STAR:EGKB:03:SUMUM1C:SUMUM LOGAN JACKO NONVA BABKU LCE21 LCE22 LCE23 RAVSA GAPGI ATPEV LCE07 OSVEV
 STAR:EGKB:21:XAMAN1C:XAMAN LOGAN JACKO NONVA BABKU LCE21 LCE22 LCE23 RAVSA GAPGI ATPEV LCE07 OSVEV
 STAR:EGKB:03:XAMAN1C:XAMAN LOGAN JACKO NONVA BABKU LCE21 LCE22 LCE23 RAVSA GAPGI ATPEV LCE07 OSVEV
-;Stack Swap Stars
-STAR:EGKB:21:GODLU1K:XAMAN TEVMO GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
-STAR:EGKB:03:GODLU1K:XAMAN TEVMO GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
-STAR:EGKB:21:GODLU1L:SUMUM TEVMO GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
-STAR:EGKB:03:GODLU1L:SUMUM TEVMO GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
+;Transitions
 STAR:EGKB:21:OSVEV1G:GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
 STAR:EGKB:03:OSVEV1G:GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
 STAR:EGKB:21:OSVEV1J:JACKO NONVA BABKU LCE21 LCE22 LCE23 RAVSA GAPGI ATPEV LCE07 OSVEV
 STAR:EGKB:03:OSVEV1J:JACKO NONVA BABKU LCE21 LCE22 LCE23 RAVSA GAPGI ATPEV LCE07 OSVEV
+;Stack Swap Stars
+STAR:EGKB:21:SUMUM1X:SUMUM TEVMO GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
+STAR:EGKB:03:SUMUM1X:SUMUM TEVMO GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
+STAR:EGKB:21:XAMAN1X:XAMAN TEVMO GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
+STAR:EGKB:03:XAMAN1X:XAMAN TEVMO GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
 ;Withdrawn 2018/13
+STAR:EGKB:21:#GODLU1K:XAMAN TEVMO GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
```

Comment:
Can be pretty certain these have never been used (Thames is never busy enough for stack swap), so I'd just remove these completely rather than archiving them! :)